Extensions/timestamps.js

```diff
@@ -16,46 +16,46 @@ XKit.extensions.timestamps = new Object({
 	slow: true,
 
 	preferences: {
-		only_on_hover: {
-			text: "Only show timestamps when I hover over a post",
-			default: false,
-			value: false
+		inbox: {
+			text: "Show timestamps in the inbox",
+			default: true,
+			value: true
 		},
-		only_relative: {
-			text: "Only show relative time (eg: 5 minutes ago)",
-			default: false,
-			value: false
+		posts: {
+			text: "Show timestamps on posts",
+			default: true,
+			value: true
+		},
+		op: {
+			text: "Show timestamps on reblogged original posts",
+			default: true,
+			value: true
 		},
-		only_inbox: {
-			text: "Only show timestamps on asks in my inbox",
+		reblogs: {
+			text: "Show timestamps on reblog comments",
 			default: false,
 			value: false
 		},
 
-		title_reblogs: {
-			text: "Reblog timestamps",
+		display_title: {
+			text: "Display options",
 			type: "separator"
 		},
-		do_reblogs: {
-			text: "Show timestamps on reblog comments",
-			default: true,
-			value: true
-		},
-		only_original: {
-			text: "Only put timestamps on original comments",
+		show_absolute: {
+			text: "Display the absolute time",
```

Comment:
I think "show_relative" is probably still the right option here—something like "Display timestamps in relative form", default false.

Reply:
also we can just keep the old name if we're keeping the old polarity